scripts/train/yamls/finetune/gpt2-arc-easy-cpu-streaming-dataset.yaml

```diff
@@ -24,9 +24,11 @@ train_loader:
   name: finetuning
   dataset:
     ############
-    remote: ${data_remote}
-    local: ${data_local}
-    split: train
+    streams:
+      0:
```

Comment:
lets name this something just for clarity that it doesn't have to be a int. maybe just `my_data`